src/main/scala/de/heikoseeberger/sbtheader/HeaderPlugin.scala

```diff
@@ -80,31 +80,14 @@ object HeaderPlugin extends AutoPlugin {
         )
     }
 
-    object HeaderCommentStyleMapping {
-
-      val javaBlockComments: (String, CommentStyle) =
-        "java" -> CStyleBlockComment
-
-      val scalaBlockComments: (String, CommentStyle) =
-        "scala" -> CStyleBlockComment
-
-      def createFrom(
-          license: License,
-          mappings: Seq[(String, CommentStyle)] = Vector(javaBlockComments, scalaBlockComments)
-      ): Map[String, (Regex, String)] =
-        mappings.map {
-          case (headerPattern, commentStyle) => headerPattern -> commentStyle(license)
-        }(breakOut)
-    }
-
     val headerLicense: SettingKey[Option[License]] =
       settingKey(
         "The license to apply to files; None by default (enabling auto detection from project settings)"
       )
 
     val headerMappings: SettingKey[Map[String, CommentStyle]] =
       settingKey(
-        "CommentStyles to be used by file extension they should be applied to; empty by default"
+        "CommentStyles to be used by file extension they should be applied to. Default is to apply C style block comments to Scala and Java files."
```

Comment:
... be applied to; C style block comments for Scala and Java files by default
(mit Strichpunkt und ohne Punt am Ende)